aya/src/programs/lirc_mode2.rs

```diff
@@ -132,7 +132,7 @@ impl LircLink {
 
     /// Get ProgramInfo from this link
     pub fn info(&self) -> Result<ProgramInfo, ProgramError> {
-        match bpf_obj_get_info_by_fd(self.prog_fd) {
+        match bpf_prog_get_info_by_fd(self.prog_fd) {
             Ok(info) => Ok(ProgramInfo(info)),
             Err(io_error) => Err(ProgramError::SyscallError {
                 call: "bpf_obj_get_info_by_fd".to_owned(),
```

Comment:
you need to update the call attribute literal here and in other places

Reply:
Done